hw1/akinunal.js

```diff
@@ -0,0 +1,122 @@
+// 1. Write a Javascript program to create a new string which is n copies of a given string where n is a non-negative integer.
+
+console.log('------ Question 1: ------')
+
+let stringCreator = (str, num) => {
+    let strCopy = '';
+
+    if (num < 0) {
+        console.log('Error: Number is negative!');
+        return
+    }
+
+    for (let i = 0; i < num; i++) {
+        strCopy += str
+        console.log(strCopy)
```

Comment:
I'd rather do return instead of just logging because I might want to use the result of this function elsewhere: 
```
let stringCreator = (str, num) => {
  // all your code...
  return strCopy
}
const repeatingStr = stringCreator('a', 5);
console.log(repeatingStr);
```